stl/inc/regex

```diff
@@ -1187,7 +1187,8 @@ _NODISCARD bool operator!=(const match_results<_BidIt, _Alloc>& _Left, const mat
 }
 #endif // !_HAS_CXX20
 
-_INLINE_VAR constexpr unsigned int _BRE_MAX_GRP = 9U;
+_INLINE_VAR constexpr unsigned int _BRE_MAX_GRP   = 9U;

```

Comment:
Now that you're consuming at most 1 digit, do we still need the only usage of `_BRE_MAX_GRP` here?

https://github.com/microsoft/STL/blob/89ca073a866d24db42b3f4c5e406be4b28a148f4/stl/inc/regex#L4343-L4346

Reply:
I guess it effectively only defends against questionable implementations of `value` method provided by the traits class now. That doesn't seem like a reason good enough to keep it, so changed in the next commit.